sample/Network/README.md

```diff
@@ -0,0 +1,131 @@
+   Generating keys for random seed b'A59c0EFB9cD7Eccdd4483a3BFbd36EB5'
+   Init local keys for client-stack
+   Public key is H3oLLToN9Wy1Yb9R9EMZXot8xCTnQLWMiSHRCpQm9fRD
+   Verification key is EktpqGnexWaiQyr9vcXDTgNYwqT8cxmAfnX8N7qWwEcC
+   Init local keys for node-stack
+   Public key is H3oLLToN9Wy1Yb9R9EMZXot8xCTnQLWMiSHRCpQm9fRD
+   Verification key is EktpqGnexWaiQyr9vcXDTgNYwqT8cxmAfnX8N7qWwEcC
+   BLS Public key is 13U7tXXXRTLeavMEQk7MqECuKkuFrHPAwidf2cVqhaJoABmHc4SBMXHVJJkc1pJNvjLu894UZ6pSt3aAYZ5nQrfkuqbBUEToWb5vZSLHTTNnznkzx5PStPFSZkYUuA4bYNLk5b8GbwrHFKjrjqzCdjEWs2hDipAmXfd9NBh3BTEwAxS
+   Proof of possession for BLS key is RHeKLLefbDdgBMpZ9AUrS8EHPDRnFXNiJ1z8LUgqPa8eUGyeRkAR2ppPkYqcLc9ekzG8cYZMTGx8y52sZ1q2QWqs3BYBH2i3H2WxL4icRq9Kj4kqs3BQadtPWBSq4vEaWTwwieuUXFYqpvk1ALCSNmS9NmMYXYyTL8uzrstviomjXm
+
+   $ init_indy_node Steward4 0.0.0.0 9707 0.0.0.0 9708
+   Node-stack name is Steward4
+   Client-stack name is Steward4C
+   Generating keys for random seed b'F20fc06eab86A896A6Ae5D8AfEA46B68'
+   Init local keys for client-stack
+   Public key is GJVyfv4XXGHYDYmiifu8XmXyTx9jGb39hACRU23rT9Ww
+   Verification key is 8XFTwX3rHVUBddyruNTzKnBdbFqWz8eZPRasyySLD7Sv
+   Init local keys for node-stack
+   Public key is GJVyfv4XXGHYDYmiifu8XmXyTx9jGb39hACRU23rT9Ww
+   Verification key is 8XFTwX3rHVUBddyruNTzKnBdbFqWz8eZPRasyySLD7Sv
+   BLS Public key is oy7vASnhYAYo9fV1MzFSeCHEmyd2dQze6dmwWd5unwoySsA2UauUaKpV6QqwL9WQzQYRXZAoDT9jXGWwGFgCKWKVinFPj2TU5qsqAFt6PcXxQ7ZpBMEiUhQreqQv9BQsb7Upx9cNZKm4wKRyjCryX3TELb3xzz51wwsdeY8hduAKvb
+   Proof of possession for BLS key is QsrUH1e5zsdiEGij1NeY9S7CwzUdU2rzjskHNGHCQ8rtgYZyBC99MgRPzgkJHP86nWQUo2fSRvyWLQdBwvWfNtSqUBQgVScQPHg9CJXWWohWnzSP4ViBo8EEeGXEoP2NPeRnFCCfuhYAC7stZgBATFyvdFRwG58ws76qQQQsfDDHBV
+   ```
+
+6. Fill the spreadsheet.
+   
+7. Download the script from [https://github.com/sovrin-foundation/steward-tools/tree/master/create_genesis] and generate the genesis file.
+
+   ```
+   $ python genesis_from_files.py --trustees Trustees.csv --stewards Stewards.csv 
+   DEBUG:root:new line check for file: ./pool_transactions_genesis
+   INFO:root:Starting ledger...
+   INFO:root:Recovering tree from transaction log
+   INFO:root:Recovered tree in 0.002561586000410898 seconds
+   DEBUG:root:new line check for file: ./domain_transactions_genesis
+   INFO:root:Starting ledger...
+   INFO:root:Recovering tree from transaction log
+   INFO:root:Recovered tree in 0.000322740000228805 seconds
+   ```
+8. Edit `/etc/indy/indy_config.py` and change the network name.
+   `NETWORK_NAME = "newNetwork"`
+
+9. `mkdir /var/lib/indy/newNetwork`
+
+10. `cp domain_transactions_genesis /var/lib/indy/newNetwork/ && cp pool_transactions_genesis /var/lib/indy/newNetwork/`
+
+11. Start the nodes:
+
+   ```
+   start_indy_node Steward1 0.0.0.0 9701 0.0.0.0 9702
+   start_indy_node Steward2 0.0.0.0 9703 0.0.0.0 9704
+   start_indy_node Steward3 0.0.0.0 9705 0.0.0.0 9706
+   start_indy_node Steward4 0.0.0.0 9707 0.0.0.0 9708     
```

Comment:
This method of starting nodes is for provisional (`test`) networks.  The preferred method for production level networks is documented in the `Steward Validator Preparation Guide v3` linked in the reference document `Create New Indy Network`.  However, since, for the purposes of the demo, the nodes are all running on the same machine you sort of have to do it this way.